### PR TITLE
[zh-cn] Init the translation of `Origin Private File System`

### PR DESCRIPTION
**Pull request by BSPR0002**
### Description

Init the translation of `Origin Private File System`
Fix wrong translation in `File System API`

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Some comments on this

File: files/zh-cn/web/api/file_system_api/origin_private_file_system/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+扩展自[文件系统 API](/zh-CN/docs/Web/API/File_System_API) 的[文件系统访问 API](https://wicg.github.io/file-system-access/) 使用选择器提供了对文件的访问能力。例如：
+
+1. {{domxref("Window.showOpenFilePicker()")}} 允许用户选择一个文件用于访问，文件将作为结果以一个 {{domxref("FileSystemFileHandle")}} 对象的形式被返回。
+2. 调用 {{domxref("FileSystemFileHandle.getFile()")}} 以访问文件的内容，使用 {{domxref("FileSystemFileHandle.createWritable()")}} / {{domxref("FileSystemWritableFileStream.write()")}} 来修改内容。
+3. 调用 {{domxref("FileSystemHandle.requestPermission()", "FileSystemHandle.requestPermission({mode: 'readwrite'})")}} 来请求用户的权限以保存更改。
+4. 如果用户接受了权限请求，更改就会保存回原文件。
+
+这个方法可行，但是有一些限制。这些更改是对用户可见的文件系统进行的，所以会有很多适当的安全性检查（比方说 Chrome 的[安全浏览](https://developers.google.com/safe-browsing)）来防止恶意内容被写入到文件系统。这些写入不是原地的，会先写入到一个临时文件。除非通过了所有的安全性检查，否则原文件不会被修改。
+
+因此，这些操作会相当缓慢。在你进行小规模的文本时没那么明显，但是当进行像 [SQLite](https://www.sqlite.org/wasm) 数据库更改这样的更重要、更大规模的文件更新时就会遇到性能问题。
```
Comment (BSPR0002) [inline files/zh-cn/web/api/file_system_api/origin_private_file_system/index.md:21]:
```suggestion
因此，这些操作会相当缓慢。在你进行小规模的文本更新时没那么明显，但是当进行像 [SQLite](https://www.sqlite.org/wasm) 数据库更改这样的更显著、更大规模的文件更新时就会遇到性能问题。
```

Comment (BSPR0002) [inline files/zh-cn/web/api/file_system_api/origin_private_file_system/index.md:21]:
```suggestion
因此，这些操作会相当缓慢。在你进行小规模的文本更新时没那么明显，但是当进行像 [SQLite](https://www.sqlite.org/wasm) 数据库更改这样的更显著、更大规模的文件更新时就会遇到性能问题。
```

**Review by yin1999 — APPROVED** [review]
Thanks 🙏

**Pull request merged**